docs/observability/index.asciidoc

```diff
@@ -0,0 +1,24 @@
+[chapter]
+[role="xpack"]
+[[observability]]
+= Observability
+
+Observability in {kib} enables you to add and monitor your logs, system
+metrics, uptime data, and application traces, as a single stack. 
+
+On the *Observability Overview* page you have:
+
+* A central place to add and configure your data sources.
+* A variety of charts displaying analytics relating to each data source.
+* *View in app* options to drill-down and anaylze data in the {logs-app}, {metrics-app}, {uptime-app}, and the APM app. 
+* An alerts chart to keep you informed of any issues that you may need to resolve quickly.
+
+[role="screenshot"]
+image::observability/images/observability-overview.png[Observability Overview in {kib}]
```

Comment:
Since the nav bar is under construction for the next few releases, could you remove it from the screenshot?

Reply:
@KOTungseth  Sorry, do you mean the nav bar listing resources and the what's new content?